doc/staging.md

```diff
@@ -132,11 +123,10 @@ $ kci user whoami
 
 ## Storage
 
-File storage is managed separately from the API services.  As part of the Early
-Access phase, an AzureFiles token is provided to each user to be able to upload
+File storage is managed separately from the API services. An AzureFiles token is provided to each user to be able to upload
 artifacts.  It is one of the storage solutions supported by KernelCI, you may
 also use SSH or soon any S3-compatible storage.  There is a quota, currently
-5TiB for all the Early Access files.  Old files will be deleted after a while
+5TiB for all the staging files.  Old files will be deleted after a while
```

Comment:
Hi @nuclearcat 
I am not sure if this part is true for staging configs.
Could you please provide some insights?

Reply:
We dont have any policy right now (by request of @padovan ), still have to decide on that after LPC.

Reply:
Okay, then should I discard this?

Reply:
yes

Reply:
Okay, thanks. As per our discussion today, I have dropped `storage` section and added details related to accessing staging containers.